client/my-sites/stats/activity-log/index.jsx

```diff
@@ -442,6 +483,18 @@ class ActivityLog extends Component {
 			/>
 		);
 
+		const backupConfirmDialog = requestedBackup && (
+			<ActivityLogConfirmDialog
+				applySiteOffset={ this.applySiteOffset }
+				key="activity-backup-dialog"
+				onClose={ this.handleBackupDialogClose }
+				onConfirm={ this.handleBackupDialogConfirm }
+				timestamp={ requestedBackup.activityTs }
+				type={ 'backup' }
+				icon={ 'cloud-download' }
+			/>
```

Comment:
is there a need for this confirm dialog to appear to high in the React component hierarchy? I can understand how it can be frustrating to be passing props so far around, we have an awful problem in Simplenote Electron where props go _through_ other components and aren't even used.

what I'm wondering is twofold:
 - are the dialogs actually local UI concerns or could they exist as a property of global state
 - must we define the action handlers here or could we imagine creating them closer to where this dialog appears

I think that if we can find a way to move this JSX deeper and closer to where it's used it won't appear to awkward to reuse for multiple dialogs - after all, what is it but a few graphics, strings, and a couple of buttons?